proto/proto/view.proto

```diff
@@ -24,7 +24,7 @@ service ViewProtocol {
     rpc StatusStream(StatusStreamRequest) returns (stream StatusStreamResponse);
 
     // Queries for notes that have been accepted by the chain.
-    rpc Notes(NotesRequest) returns (stream NoteRecord);
+    rpc Notes(NotesRequest) returns (stream SpendableNoteRecord);
```

Comment:
Let's rename the RPC to `rpc SpendableNotes(SpendableNotesRequest)`, for consistency?